File: tailo_realm_parser_app/src/main/java/com/taccotap/taigidictparser/TaigiDictParserApp.java
package com.taccotap.taigidictparser;

import android.app.Application;

import io.realm.Realm;
import io.realm.RealmConfiguration;

public class TaigiDictParserApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        initRealm();
    }

    private void initRealm() {
        Realm.init(this);
        RealmConfiguration realmConfig = new RealmConfiguration.Builder().build();
//        Realm.deleteRealm(realmConfig); // Delete Realm between app restarts.
        Realm.setDefaultConfiguration(realmConfig);
    }
}

File: tailo_realm_parser_app/src/main/java/com/taccotap/taigidictparser/TaigiDictParserApp.kt
package com.taccotap.taigidictparser

import android.app.Application

import io.realm.Realm
import io.realm.RealmConfiguration

class TaigiDictParserApp : Application() {

    override fun onCreate() {
        super.onCreate()

        initRealm()
    }

    private fun initRealm() {
        Realm.init(this)
        val realmConfig = RealmConfiguration.Builder().build()
        //        Realm.deleteRealm(realmConfig); // Delete Realm between app restarts.
        Realm.setDefaultConfiguration(realmConfig)
    }
}
